manipulation/exercises/grader.py

```diff
@@ -57,6 +57,12 @@ def locals_from_notebook(notebook_ipynb):
         ipynb = json.load(open(notebook_ipynb))
 
         for cell in ipynb['cells']:
+            # For now, clearOutputs is unsupported, so we filter out those lines
```

Comment:
it sounds like you are filtering out the code.  could you make this comment more clear that you are filtering out the warning that gets printed to the output cell?  (for when we look at this again next time and start scratching our heads?)

otherwise, LGTM.

Reply:
pushed an update!